core/optaplanner-core-impl/src/main/java/org/optaplanner/core/impl/domain/solution/cloner/BooleanFieldCloner.java

```diff
@@ -3,16 +3,16 @@
 import java.lang.reflect.Field;
 import java.util.function.Consumer;
 
-final class BooleanFieldCloner<C> implements FieldCloner<C> {
+final class BooleanFieldCloner implements FieldCloner {
 
     private static final FieldCloner INSTANCE = new BooleanFieldCloner();
 
-    public static <C> FieldCloner<C> getInstance() {
+    public static FieldCloner getInstance() {
```

Comment:
Since the interface is no longer generic, the `getInstance()` method can be removed and we can use the `INSTANCE` field directly.
This applies to all the cloner implementations.